app/browser/api/ledger.js

```diff
@@ -3123,6 +3126,53 @@ const activityRoundTrip = (err, response, body) => {
   updater.checkForUpdate(false, true)
 }
 
+const deleteWallet = (state) => {
+  state = ledgerState.deleteSynopsis(state)
+  state = state.setIn(['settings', settings.PAYMENTS_ENABLED], false)
```

Comment:
I don't think that this is needed, because we will delete all data related to the payments in the state, so when you run it again it should override this file if still exists